canonical-web-and-design/canonical.com
# Update partner programmes

## Done

- Prepare "Partners" section for release

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8002/ and click on `Partners` in the navigation
- Navigate to http://0.0.0.0:8002/partners and see the available partner programmes. QA the links on this page
- QA all the links on the following pages: http://0.0.0.0:8002/partners/public-cloud, http://0.0.0.0:8002/partners/ihv-and-oem, http://0.0.0.0:8002/partners/desktop, http://0.0.0.0:8002/partners/channel-and-reseller, http://0.0.0.0:8002/partners/gsi & http://0.0.0.0:8002/partners/become-a-partner


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2559 & https://github.com/canonical-web-and-design/canonical.com/issues/214

## Screenshots

[if relevant, include a screenshot]
